src/plugins/GenerateJob/templates/main.ejs

```diff
@@ -22,7 +22,7 @@ signal.signal(signal.SIGTERM, signal_handler)
 <%= name %> = result
 ############ <%= name %> created ############<%}); %>
 
-from operations import <%= name %>
+from operations.<%= filename %> import <%= name %>
```

Comment:
Not sure if we should need to change the style of imports to resolve this issue.

Reply:
If we follow suggestions from your previous comments. It wouldn't be necessary.